tests/ast/test_ast_driver.py

```diff
@@ -110,8 +119,20 @@ def run_test(session, test_name, test_source):
 from snowflake.snowpark.functions import *
 from snowflake.snowpark.types import *
 from snowflake.snowpark import Table
+from snowflake.snowpark._internal.ast import AstBatch
+import snowflake.snowpark._internal.ast_utils as ast_utils
+
+import uuid
+
+# Set up the request ID generator.
+AstBatch.generate_request_id = lambda: uuid.uuid5(uuid.NAMESPACE_DNS, "id-gen")
+
+ast_utils.SRC_POSITION_TEST_MODE = True
 
 def run_test(session):
+    # Reset the entity ID generator.
```

Comment:
nit: this should be a public function (or private `_reset`). Calling constructor manually sounds ... dangerous?